addons/ai/initSettings.sqf

```diff
@@ -0,0 +1,10 @@
+private _category = format ["ACE %1", localize LSTRING(DisplayName)];
+
+[
+    QGVAR(assignNVG), "CHECKBOX",
+    [LSTRING(AssignNVG_DisplayName), LSTRING(AssignNVG_Description)],
+    _category,
+    false,
+    1,
+    FUNC(assignNVG)
```

Comment:
1. Start the mission with setting enabled
2. Disabled the setting, enable the setting (before 300s passed)
3. Two loops will run

Reply:
Indeed. To be fair, should probably just require a restart for this setting. Runtime changes don't really make sense, as we don't revert to whatever it was originally anyways. Thoughts?

Reply:
Why not use a PFH with a delay of 300s? That way you could save the ID and check if it exists or not. It would allow for a CBA setting that can be changed mid mission.

On another note: Having this functionality run on unit creation would be nice imo, that way you wouldn't have to wait up to 5 mins to get an AI to have its NVGs equipped.

On yet another note: If you assign NVGs to a unit, do the AI use them automatically?

Reply:
PFH would work yes.

First run is done without a delay, could also run on new unit creation once.

AI uses them automatically if it's assigned yes. But they don't take them automatically from their uniform, which is what this PR does. It's more a cosmetic thing so you don't constantly have AI with NVGs running around, especially prevalent with vanilla factions.